Kickstarter-iOS/Views/Controllers/PledgeAmountViewController.swift

```diff
@@ -144,6 +144,10 @@ final class PledgeAmountViewController: UIViewController {
       }
   }
 
+  override func didMove(toParent parent: UIViewController?) {
+    self.verticalSpacer.isHidden = true
```

Comment:
This spacer is not needed when this view is contained and was causing the scrollview to have an ambiguous content size.